src/NetherPortalScanner.h

```diff
@@ -39,8 +39,8 @@ class cNetherPortalScanner : public cChunkStay
 	static const int SearchSolidBaseWidth = 3;
 
 	/** Where to place the player out from the face and across the face */
-	const double OutOffset = 2;
-	const double AcrossOffset = 0.5;
+	const double m_OutOffset = 2;
+	const double m_AcrossOffset = 0.5;
```

Comment:
I think these are supposed to be `static` constants and so shouldn't have the `m_` prefix.